torchqdynamics/solvers/ode/adaptive_solver.py

```diff
@@ -67,9 +67,7 @@ def run_autograd(self):
                     t, y, ft = t + dt, y_new, ft_new
 
                     # update the progress bar
-                    with warnings.catch_warnings():  # ignore tqdm precision overflow
-                        warnings.simplefilter('ignore', TqdmWarning)
-                        pbar.update(dt.item())
```

Comment:
Are you sure this one is not issuing warnings anymore? At the time it was for me.

Reply:
It does not for the current tests set. Maybe we need both, let's merge this and see if we need this one too 🤷‍♂️